lib/screens/authentication/import_key/interactor/mappers/import_key_state_mapper.dart

```diff
@@ -6,22 +6,30 @@ import 'package:seeds/screens/authentication/import_key/import_key_errors.dart';
 import 'package:seeds/screens/authentication/import_key/interactor/viewmodels/import_key_bloc.dart';
 
 class ImportKeyStateMapper extends StateMapper {
-  ImportKeyState mapResultsToState(ImportKeyState currentState, List<Result> results, AuthDataModel authData) {
+  ImportKeyState mapResultsToState({
+    required ImportKeyState currentState,
+    required AuthDataModel authData,
+    required List<Result> results,
+    AuthDataModel? alternateAuthData,
+    List<Result> alternateResults = const [],
```

Comment:
is the empty default value needed? const [],

can we make this a required field too?

Reply:
There's cases where there are no alternate results - when we import a plain private key for example. 

Import plain private key -> only results and authdata are defined, alternate undefined. I guess we could make alternate results nullable too "?".